### PR TITLE
Document new ``symfony-database-migrate`` helper

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: config.rst [hunk 1]
```diff
@@ -502,7 +502,23 @@ automation during the ``symfony-build`` run.
...
 **symfony-deploy** is to be used each time a Symfony application is deployed.
 Its purpose is to make Symfony cache used by the application if built by
-:doc:`symfony-build`, built it otherwise.
+symfony-build_, built it otherwise. When ran from the web container, it
+also runs the symfony-database-migrate_ helper.
+
+.. _symfony-database-migrate:
+
+(>&2 symfony-database-migrate)
+....................
```
Comment (OskarStark) [inline config.rst]: The underline is to short

Comment (tucksaun) [reply config.rst]: indeed!
thank you for catching this.

File: config.rst [hunk 2]
```diff
@@ -502,7 +502,23 @@ automation during the ``symfony-build`` run.
...
+symfony-build_, built it otherwise. When ran from the web container, it
+also runs the symfony-database-migrate_ helper.
+
+.. _symfony-database-migrate:
+
+(>&2 symfony-database-migrate)
+..............................
+
+You usually don't need to worry about this helper.
+**symfony-database-migrate** purpose is to runs database migrations. By
```
Comment (fabpot) [inline config.rst]: to run

File: config.rst [hunk 3]
```diff
@@ -502,7 +502,23 @@ automation during the ``symfony-build`` run.
...
+default, it will run your Doctrine migrations if
+``doctrine/doctrine-migrations-bundle`` is installed. If your application uses
+another migrations system you can override
```
Comment (fabpot) [inline config.rst]: migration system,

**Pull request merged**